src/additional/vinUS.js

```diff
@@ -43,11 +45,15 @@ $.validator.addMethod( "vinUS", function( v ) {
 		}
 		rs += d;
 	}
-	cd = rs % 11;
+	
+    // check digit is an integer
+    cd = rs % 11;
 	if ( cd === 10 ) {
-		cd = "X";
+		// check digit is a string
+        cd = "X";
 	}
-	if ( cd === cdv ) {
+	// cd and cdv can be either integer  string
+	if ( cd == cdv ) {
```

Comment:
I think `parseInt()` should be used instead. For example, `(parseInt("X") === parseInt("X")) === false`. Any complaint against that?

```suggestion
	if ( parseInt(cd) === parseInt(cdv) ) {
```

Reply:
No, that will not work, if string cd ="X" and string cdv="X" should = true.
line 50 could be 'if (+cd === +cdv')' the plus(+) on a string will evaluate it to an int, other wise it is a string.

In reality, the vin check digit in location 8 should be transliterated to a number not a string.
That way the check digit cd and check digit vin will always be a number.